### DISCUSSION TITLE
Linking to "Meetings Near Me" no longer works the same.

### DISCUSSION THREAD

**Discussion by lionssharedigital** [code4recovery/12-step-meeting-list#1546] [General]
On https://austinaa.org/, we have a button for "Upcoming Meetings Near Me". 
The query string is: https://austinaa.org/meetings/?tsml-time=upcoming&tsml-mode=me&tsml-distance=10&mode=me&distance=10
I recently added distance=10 because that appears to be needed. 

Has there been a recent change in browsers or the plugin? It looks like you the user now has to allow the page to use their location. I don't recall it working that way. 

Are you all aware of this?

**Comment by joshreisner** [comment]
the query string did change a little from the "legacy" view and the TSML UI view. i see artifacts of both in your current link. but it's always prompted the user to share their location

if you just want to make the query string `mode=me&distance=10` that's all you need now that you're using TSML UI. no need for `tsml-time=upcoming` or `tsml-mode=me`

**Reply by joshreisner** [reply]
we've got an open issue to make TSML handle this redirect automatically - we'll prioritize it: https://github.com/code4recovery/12-step-meeting-list/issues/1461
